src/shapes.ts

```diff
@@ -91,6 +91,16 @@ export interface OpenFinWindow {
     id: number;
     name: string;
     uuid: string;
+    preloadState?: { // set if window has a preload script
+        src: String;
```

Comment:
`src` should be [`PreloadOption`](https://github.com/HadoukenIO/core/blob/develop/src/browser/preload_scripts.ts#L28)

While you're at it, please change `PreloadOption` to a union type including string:
```js
type PreloadOption = string | PreloadInstance[];
```